gobblin-runtime/src/main/java/org/apache/gobblin/service/monitoring/JobStatusRetriever.java

```diff
@@ -33,14 +35,19 @@
   public abstract Iterator<JobStatus> getJobStatusesForFlowExecution(String flowName, String flowGroup,
       long flowExecutionId);
 
+  public Iterator<JobStatus> getJobStatusesForFlowExecution(String flowName, String flowGroup,
```

Comment:
Make this abstract since a reasonable implementation is required for a fully functional `JobStatusRetriever`.